site/content/docs/cli/npm-packages.md

```diff
@@ -26,4 +29,58 @@ The latest runtime is `2022.10` at the time of writing. This runtime contains am
 - [Browse the latest runtime specs](https://www.checklyhq.com/docs/runtimes/specs/)
 - [Learn more about runtimes](https://www.checklyhq.com/docs/runtimes/)
 
+## Local Dependencies
+
+Browser Checks are also able to import other JavaScript and TypeScript files as dependencies.
+This can be particularly useful for defining helper functions to be reused across multiple checks.
+The Checkly CLI will automatically detect these dependencies and make sure that they're bundled as part of the check.
+No additonal configuration is needed, and there's no need to create [Snippets](/docs/snippets/) in your account.
+
+Here is an example of how this works in practice. The directory tree looks like the following:
+```
+__checks__
+├── login.check.ts
+├── login.spec.ts
+└── login-helper.ts
+```
+
+In `login-helper.ts` we define a reusable function `gitHubLogin`. This function can be used by multiple Browser checks.
+```
```

Comment:
@clample I think we want to use syntax highlighting here and in most other code blocks